xchange-stream-core/src/main/java/info/bitrich/xchangestream/core/StreamingExchange.java

```diff
@@ -15,6 +15,7 @@
   String ENABLE_LOGGING_HANDLER = "Enable_Logging_Handler";
   String SOCKS_PROXY_HOST = "SOCKS_Proxy_Host";
   String SOCKS_PROXY_PORT = "SOCKS_Proxy_Port";
+  String DISABLE_NETTY_RECONNECT = "No_Netty_Retry";
```

Comment:
I'd prefer to not expose the Netty aspect of this to higher API's and make it more descriptive. Perhaps, "Auto_Channel_Reconnect", where a true value enables reconnecting of channels, but a false disables it?